srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/NewVarController.java

```diff
@@ -0,0 +1,110 @@
+package gov.va.med.srcalc.web.controller.admin;
+
+import java.util.Objects;
+
+import javax.validation.Valid;
+
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.Validator;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.google.common.collect.Iterables;
+
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.web.view.admin.EditVariable;
+
+/**
+ * <p>Base Web MVC controller for creating a new variable. Contains common
+ * behavior for creating a new variable, only requiring subclasses to define
+ * variable type-specific behavior.</p>
+ * 
+ * <p>Subclasses should only need to implement the abstract methods.</p>
+ * 
+ * <p>Note: normally we do not abbreviate type names, but we are abbreviating
+ * "Variable" here or else some of these controller names would be very long.</p>
+ */
+public abstract class NewVarController
```

Comment:
I'll push you off that fence. How about `NewDiscreteNumericalVariableController`?